test/Sentry.Tests/SentryClientTests.cs

```diff
@@ -612,6 +613,68 @@ public void CaptureEvent_WithSampleRate_AppropriateDistribution(float sampleRate
         });
     }
 
+    [Fact]
+    public void CaptureEvent_Processing_Order()
```

Comment:
That looks like it's been fun figuring out!

Reply:
Yes, deep in the weeds of NSubstitute!